release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1058,6 +1076,18 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-web-console-developer-perspective-bug-fixes"]
 ==== Web console (Developer perspective)
 
+* Before this update, the *Git Import* form displayed an error message when an invalid devfile repository (older than devfile v2.2) is entered. With this update, the error message state that devfiles older than v2.2 is not supported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2046435[*BZ#2046435*])
+
+* Before this update, if the *ConsoleLink CR* (openshift-blog) was not available in the cluster that causes the blog link to be undefined. So while clicking on the blog link did not redirect to the openshift blog. With this update fall back link to https://developers.redhat.com/products/openshift/whats-new is added even if the *ConsoleLink CR* (openshift-blog) is not present in the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050637[BZ#2050637])
```

Comment:
iiuc, consider the following rewording:
```suggestion
* Before this update, if the *ConsoleLink CR* (openshift-blog) was not available in the cluster, it would cause the blog link to be undefined. This would prevent a user from being redirected to the {product-title} blog if they clicked on the blog link. With this update, a fall back link to https://developers.redhat.com/products/openshift/whats-new is added even if the *ConsoleLink CR* (openshift-blog) is not present in the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050637[BZ#2050637])
```